platform/cooja/contiki-cooja-main.c

```diff
@@ -39,7 +39,8 @@
 #include <stdio.h>
 #include <string.h>
 
-#include "contiki.h"
+#include <contiki.h>
```

Comment:
According to what @adamdunkels said, double quotes should be used here, not angle brackets.